doc/dual.xml

```diff
@@ -0,0 +1,115 @@
+#############################################################################
+##
+#W  dual.xml
+#Y  Copyright (C) 2018                                             Finn Smith
+##
+##  Licensing information can be found in the README file of this package.
+##
+#############################################################################
+##
+
+<#GAPDoc Label="DualSemigroup">
+  <ManSection>
+    <Attr Name = "DualSemigroup" Arg = "sgrp"/>
+    <Returns>The dual semigroup of the given semigroup.</Returns>
+    <Description>
+      This method returns a representation of the dual semigroup of 
```

Comment:
"Returns a semigroup isomorphic to the dual semigroup of <A>S</A>." It isn't a method, it's an attribute, and it returns a semigroup isomorphic to the dual semigroup rather than a representation of the dual semigroup.